backend/task.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+from sklearn.cluster import KMeans
+import pandas as pd
+import numpy as np
+import re
+import nltk
+#import multiprocessing
```

Comment:
There are a few lines of commented out code (lines, 7, 20, 22, 54, and 56). Are these going to be used later? If not, they should be removed.

Reply:
I removed

Reply:
Thanks, remove the CSV and I'll approve!

Reply:
Thank you, I also changed the base to development from backend.